tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     clean: all
   variables:
   - name: skipComponentGovernanceDetection
```

Comment:
We may simplify this part: always set it to true. 
We manually include the other yaml file in every job. So this auto-injected Component Governance task is unnecessary.